vault/mount.go

```diff
@@ -2386,3 +2386,217 @@ func (c *Core) mountEntryView(me *MountEntry) (BarrierView, error) {
 
 	return nil, errors.New("invalid mount entry")
 }
+
+func (c *Core) invalidateMount(ctx context.Context, key string) {
+	go func() {
+		err := c.invalidateMountInternal(ctx, key)
+		if err != nil {
+			c.logger.Error("unable to invalidate mount, restarting core", "key", key, "error", err.Error())
+			// TODO(phil9909) call c.restart()
+			return
+		}
+	}()
+}
+
+func (c *Core) invalidateNamespaceMounts(ctx context.Context, uuid string) {
```

Comment:
Is this called from somewhere? :eyes:

Reply:
@phil9909 Ah, I had missed that. Already clicked "mark file as viewed" when I went searching so I didn't find it. :facepalm: